src/css/_navigation.scss

```diff
@@ -73,15 +73,25 @@
   position: relative;
   top:-4px;
 }
-
-
```

Comment:
As @psiinon mentioned earlier the white space changes should be reverted

Reply:
@Moeez905 here you actually need to bring back the white spaces that have been removed by the extension you used, `-` sign in red denotes an empty space that has been removed by one of your commits